npm/src/lib/libDef.js

```diff
@@ -346,3 +356,80 @@ export async function getShallowGHLibDefs(
   }
   return _libdefs
 }
+
+const identity = i => i
+
+function getGHFlowVersionsForDef(def: ShallowGHLibDef): Promise<Array<Version>> {
+  const getContent = Rx.Observable.fromCallback(gitHubClient().repos.getContent)
+  const notNull = i => i != null
+  return getContent({
+    user: 'flowtype',
+    repo: 'flow-typed',
+    path: `/definitions/${def.libDef.pkgNameVersionStr}`
+  })
+  .flatMap(identity)
+  .filter(notNull)
+  .flatMap(identity)
+  .map(i => i.name)
+  .filter(name => name.indexOf('flow-') == 0)
+  .map((name:string): Version => {
+    let matches = name.match(/flow-(.*)/)
+    if (matches && matches.length > 1) {
+      const withoutPrefix = matches[1]
+      return stringToVersion(withoutPrefix)
+    } else {
+      return emptyVersion()
+    }
+  })
+  .toArray()
+  .toPromise()
+}
+
+export function getGHLibsAndFlowVersions(
```

Comment:
Should probably call `_verifyCLIVersion()` in here first to be sure we're working with a version of the GH repo that we're compatible with